Add new blog post for Humans of OTel recap (#3704)
open-telemetry/opentelemetry.io: content/en/blog/2023/humans-of-otel.md

---
title: The Humans of OpenTelemetry
linkTitle: Humans of OTel
date: 2023-12-21
author: >-
  [Adriana Villela](https://github.com/avillela) (Lightstep),
# prettier-ignore
cSpell:ignore: ADIANA ALEX Alex ARONOFF Aronoff BOGDAN Bogdan BOTEN Boten CARAMANOLIS Caramanolis CONSTANCE Constance Dapr DRUTU Drutu JACOB Jacob JURACI KANAL Kanal KRÖHLING PAIXĀO Paixāo PURVI Purvi TYLER Tyler utopic VILLELA YAHN Yahn youtube
---

What a year it has been for OpenTelemetry!
[The OTel Demo turned 1](/blog/2023/demo-birthday/),

> [the OpenTelemetry project announced general availability of the OpenTelemetry specification](https://www.apmdigest.com/opentelemetry-announces-general-availability),

Author: Ooh, I like this even better...I was looking for something like this - thank you!

[Trace-Based Testing was added to the OTel Demo](/blog/2023/testing-otel-demo/),
we saw some [exciting OTel integrations](/blog/2023/otterize-otel/), and let's
not forget how we had not one, but TWO Observability Days in 2023 - one for

> [KubeCon Europe in Amsterdam](https://shorturl.at/osHRX), and one for
> [KubeCon North America in Chicago](https://shorturl.at/kAEQX).

Reviewer: ```suggestion
[KubeCon Europe in Amsterdam](https://www.youtube.com/playlist?list=PLj6h78yzYM2ORxwcjTn4RLAOQOYjvQ2A3), and one for
[KubeCon North America in Chicago](https://www.youtube.com/playlist?list=PLj6h78yzYM2N60fCRNLBL7ymUHg7fFbDZ).
```

Is there an advantage of using the shorturl over the direct links to YouTube?

None of this would've been possible without the awesome humans behind
OpenTelemetry. Whether you're a maintainer, contributor, or practitioner, we
want y'all to know that what you do matters!

And so, to wrap up 2023, I interviewed some of the folks who have been involved
with OTel, both past and present:

> - [Tyler Yahn](https://www.linkedin.com/in/tyler-yahn/)
> - [Amy Tobey](https://www.linkedin.com/in/amytobey/)
> - [Ted Young](https://www.linkedin.com/in/ted-young/)
> - [Carter Socha](https://www.linkedin.com/in/cartersocha/)
> - [Bogdan Drutu](https://www.linkedin.com/in/bogdan-drutu-7909602a/)
> - [Constance Caramanolis](https://www.linkedin.com/in/ccaramanolis/)
> - [Juraci Paixāo Kröhling](https://www.linkedin.com/in/jpkroehling/)
> - [Jacob Aronoff](https://www.linkedin.com/in/jaronoff97/)
> - [Alex Boten](https://www.linkedin.com/in/codeboten/)
> - [Purvi Kanal](https://www.linkedin.com/in/🚀-purvi-kanal-4a36089a/)

Reviewer: ```suggestion
- [Tyler Yahn](https://github.com/MrAlias/)
- [Amy Tobey](https://github.com/tobert)
- [Ted Young](https://github.com/tedsuo)
- [Carter Socha](https://github.com/cartersocha)
- [Bogdan Drutu](https://github.com/bogdandrutu)
- [Constance Caramanolis](https://github.com/ccaraman)
- [Juraci Paixāo Kröhling](https://github.com/jpkrohling)
- [Jacob Aronoff](https://github.com/jaronoff97)
- [Alex Boten](https://github.com/codeboten)
- [Purvi Kanal](https://github.com/pkanal)
```

Special thanks to [Reese Lee](https://www.linkedin.com/in/reese-lee/) for the
camera work!

You can watch the full recording here:

{{<youtube coPrhP_7lVU>}}

<br/>Thanks to everyone who has contributed to OpenTelemetry to date, and we
look forward to your contributions in 2024! 🎉

## Transcript

If reading is more your jam, check out the transcript of our conversations
below.

### 1- Meet the Humans of OTel

**TYLER YAHN:** I'm Tyler Yahn. I am a maintainer for the OpenTelemetry Go SIG.

> We're working on some auto-instrumentation there, and specification.

Reviewer: ```suggestion
We're working on some [auto-instrumentation](https://github.com/open-telemetry/opentelemetry-go-instrumentation/) there, and [specification](https://github.com/open-telemetry/opentelemetry-specification/).
```

**AMY TOBEY:** I'm Amy Tobey. I am senior principal engineer for digital
interconnection at Equinix. I maintain a tool called OTel CLI.

**ADRIANA VILLELA:** Oh, you maintain the OTel CLI!

**AMY TOBEY:** Yeah, that's my project. Yeah. So that mostly just has Traces
right now. And I've been meaning to implement Logs and Metrics for a while and I
think like Logs just went GA recently, so it's time to do it. But Traces have
been so effective and people really like it that I haven't really had a lot of
demand for them.

**ADRIANA VILLELA:** Is the OTel CLI part of OpenTelemetry?

**AMY TOBEY:** It's not yet. I maintain it on our Equinix Labs GitHub account.
It's not a lot of process. Mostly it's just me with a few folks like Alex and
others that throw me a PR every now and then. But I've thought about bringing it
back to the community. But I'd have to maybe be not as so far away from the
standards as I am right now because doing it in the command line, a lot of the
standards don't really translate very well. So I've strayed a little bit away
from the standards in a few places. It would make sense. And I've talked to
Austin about it.

> **ADRIANA VILLELA:** Hey, I've got Ted Young with me! Hello!

Reviewer: ```suggestion
**ADRIANA VILLELA:** Hey, I've got Ted Young with me! Hello!

**TED YOUNG:** Hello!
```

I think I heard Ted say Hello, so it looks better instead of having Carter speak next

**CARTER SOCHA:** I'm Carter Socha. I work on a couple different things. I'm one
of the few product managers floating around, but I helped start the

> OpenTelemetry Demo, which I'm a maintainer of. I also work in the SIG security,

Reviewer: ```suggestion
[OpenTelemetry Demo](/docs/demo/), which I'm a maintainer of. I also work in the [SIG security](https://github.com/open-telemetry/sig-security),
```

which helps the project improve their security response process.

**BOGDAN DRUTU:** My name is Bogdan. I took a break for parental leave so I'm
just jumping back. Okay, what were you doing before? I done a lot of things,

> including member of TC, member of GC, maintainer of Collector. I was a former

Reviewer: ```suggestion
including member of [TC](https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md), member of [GC](https://github.com/open-telemetry/community/blob/main/governance-charter.md), maintainer of [Collector](https://github.com/open-telemetry/opentelemetry-collector). I was a former
```

> maintainer of Java, so I've done a lot.

Reviewer: ```suggestion
maintainer of [Java](https://github.com/open-telemetry/opentelemetry-java), so I've done a lot.
```

**CONSTANCE CARAMANOLIS:** Hi. I'm Constance Caramanolis.

**ADRIANA VILLELA:** I know that you were involved in OpenTelemetry and you are
one of the OG contributors. Tell us about that involvement.

**CONSTANCE CARAMANOLIS:** Yeah, so I worked on the OpenTelemetry Collector. I
contributed to that. I did a lot of config things. I was also on the
OpenTelemetry Governance Committee. So I did a lot of the start, we were doing
the incubation process, starting a whole gathering process, a POC, a lot of
putting processes in place, getting adoption. Quite a few talks. KubeCon
talks...

**JURACI PAIXĀO KRÖHLING:** My name is Juraci. I'm a software engineer and I've
been working with OpenTelemetry systems or Observability for a few years now. I

> come from a Tracing background, so I was a maintainer on Jaeger. I was part of

Reviewer: ```suggestion
come from a Tracing background, so I was a maintainer on [Jaeger](https://www.jaegertracing.io/). I was part of
```

OpenTracing back in the day and I helped choose the name of the project that we
have. And right now I'm a Collector developer. I help out on some components for
OpenTelemetry Collector. And I'm also part of the Governing Committee for
OpenTelemetry.

> **ADRIANA VILLELA:** And newly re-elected, right? I was just re-elected, yes.

Reviewer: ```suggestion
**ADRIANA VILLELA:** And newly re-elected, right? 

**JURACI PAIXĀO KRÖHLING:** I was just re-elected, yes.
```

I think this needs to be changed like this?

**JACOB ARONOFF:** My name is Jacob Aronoff. I am a maintainer for the

> OpenTelemetry Operator project.

Reviewer: ```suggestion
[OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator) project.
```

**ALEX BOTEN:** Hi, I'm Alex and I'm a contributor and maintainer in

> OpenTelemetry. I Wrote a book about OpenTelemetry. I don't know what else. I do

Reviewer: ```suggestion
OpenTelemetry. I wrote a book about OpenTelemetry. I don't know what else. I do
```

stuff with OTel. Cool. I am a contributor and maintainer of the OpenTelemetry

> Collector and the OpenTelemetry Collector Contrib repository and I have been

Reviewer: ```suggestion
Collector and the [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) repository and I have been
```

spending a lot of time in various SIGs and specialty working groups around

> configuration and security. And previously I spent a bunch of time maintaining

Reviewer: ```suggestion
[configuration](https://github.com/open-telemetry/opentelemetry-configuration) and security. And previously I spent a bunch of time maintaining
```

> and contributing to Python.

Reviewer: ```suggestion
and contributing to [Python](https://github.com/open-telemetry/opentelemetry-python).
```

**PURVI KANAL:** Hey, my name is Purvi. I am a senior software engineer. I
worked over my career a lot with browsers and JavaScript.

### 2- What does Observability mean to you?

**TYLER YAHN:** Yeah, that's a great question. Personally, I think Observability

> means that when you woke up at 2:00am To go fix a problem, you can fix it. And

Reviewer: ```suggestion
means that when you woke up at 2:00am to go fix a problem, you can fix it. And
```

ideally, the next day you're able to look at that code again and find out a way
to never have that problem exist. I think that's really what it means to me.

**AMY TOBEY:** It means being able to look at things coming out of the box and
tell what's going on inside parts. Very convenient.

> **TED YOUNG:** First of all, it's monitoring...But really, Observability is this

Reviewer: ```suggestion
**TED YOUNG:** First of all, it's monitoring… But really, Observability is this
```

try to whittle it down to something. I'm going to look at all the config files
myself, try to figure out what's going on. You can just quickly get an answer to
a lot of those questions and then move on to the next hypothesis.

The amount of time you save with modern Observability, I think, changes how we
actually practice, and that's an ongoing trend. But with OpenTelemetry going,
effectively going GA this year, with tracing, metrics and logs now stable, yes,
finally, only like two years late anyway. But the fact that we have that now,
the fact that we now have telemetry that has all of these correlations baked
into it, you're going to start seeing a new wave of analysis tools, all the
existing ones out there, but also new ones being built, that leverage the fact
that this data is available and that it's like a standard data format, kind of
proprietary data format, stable data format. You can rely on it.

So it's like okay to build your giant platform on top of this data or build some
kind of like boutique analysis tool that just does one thing and does it really
well. That's where I see it all going. And that's what Observability means to
me.

> **CARTER SOCHA:** What does Observability mean to me... It means like, an

Reviewer: ```suggestion
**CARTER SOCHA:** What does Observability mean to me… It means like, an
```

application owner can see what's going on in their environment, and answer
pertinent questions to them about their business and how they can improve their
service.

**BOGDAN DRUTU:** Observability is an overloaded term in our days, but it means
the capability of monitoring and determining when something goes wrong in your
production environment.

> **CONSTANCE CARAMANOLIS:** Observability means...what does it mean to me? I use

Reviewer: ```suggestion
**CONSTANCE CARAMANOLIS:** Observability means… what does it mean to me? I use
```

And then I got switched to working with a development team that was focused
solely on OpenTelemetry, both for external purposes and internal purposes,
because Microsoft uses OpenTelemetry really heavily internally. And so that's
what got me introduced. And when I started looking around, wondering where I
could start, I realized there was no real good example of how to use
OpenTelemetry in the wild. And so that was a problem that I thought every vendor
might have. And something we could solve together as a community, and we have.

**ADRIANA VILLELA:** How long have you been working on OpenTelemetry?

**BOGDAN DRUTU:** Since the beginning.

**ADRIANA VILLELA:** So like 2019 or like, pre?

**BOGDAN DRUTU:** Even pre.

**ADRIANA VILLELA:** Did you start out with Ted in the...pre days?

**BOGDAN DRUTU:** No. Actually, there were two competing projects that merged
into OpenTelemetry.

**ADRIANA VILLELA:** Right.

**BOGDAN DRUTU:** So I was on the other project.

**ADRIANA VILLELA:** Which one, OpenCensus?

**BOGDAN DRUTU:** Yeah.

**PURVI KANAL:** I got involved with OpenTelemetry through working at Honeycomb.

> So I got involved with it, and I have a particular interest in OpenTelemetry
> JavaScript, and especially the browser side of OpenTelemetry JavaScript. It's

Reviewer: ```suggestion
So I got involved with it, and I have a particular interest in [OpenTelemetry JavaScript](https://github.com/open-telemetry/opentelemetry-js), and especially the browser side of OpenTelemetry JavaScript. It's
```

on a daily basis for a few years. Yeah. That's what OpenTelemetry is for me.

**JACOB ARONOFF:** OpenTelemetry is Observability backed by everybody. It's not
a single vendor. It's letting you do the thing that is agnostic to where you
send the data. In the same way that you don't have to relearn how to drive a car
every time you step into a new car. You don't have to learn how to ride a bike
based on the vendor of the bike that you buy from. You should be able to
instrument your code no matter where you send that data. So that's how I sell
it. That's how I think about it.

**JACOB ARONOFF:** The other benefit is we as the maintainers, we have a lot of
our maintainers here and approvers here, so we can collaborate and work together
to figure out what's really needed in the next coming months. I described it
almost like summer camp. There are some people where, oh, I haven't seen you in
a few months. How you been? It's catching up.

**ALEX BOTEN:** Like, I mean, OTel has been amazing. The project itself has been
wonderful. It's one of the first projects to take a bunch of standards and
condense them down into less standards. We took OpenCensus, OpenTracing, we

> brought Prometheus to the table. The Elastic Cache format is there.

Reviewer: ```suggestion
brought [Prometheus](https://prometheus.io/) to the table. The [Elastic Cache format](/blog/2023/ecs-otel-semconv-convergence/) is there.
```

---
title: The Humans of OpenTelemetry
linkTitle: Humans of OTel

> date: 2023-12-21

Reviewer: ```suggestion
date: 2023-12-22
```
